### PR TITLE
Fix alternate upgrade method of XCP-ng

### PR DESCRIPTION
**Pull request by benjamreis**
The plugin `prepare_host_upgrade.py` looks for the added entries in the `.treeinfo` in order to do the upgrade.

### CODE REVIEW

**Review by gduperrey — APPROVED** [review]

File: templates/iso/8.3/.treeinfo [hunk 1]
```diff
@@ -1,3 +1,11 @@
+[platform]
+name = @@PLATFORM_NAME@@
+version = @@PLATFORM_VERSION@@
+
+[branding]
+name = @@PRODUCT_BRAND@@
+version = @@PRODUCT_VERSION@@
+
```
Comment (ydirson) [inline templates/iso/8.3/.treeinfo:8]: Those should at least be flagged with as "legacy" using a comment.

**Review by ydirson — CHANGES_REQUESTED** [review]
Those `.treeinfo` are modeled on the XS ones, is this upgrade method something that XS does not support?

**Comment by benjamreis** [comment]
> Those `.treeinfo` are modeled on the XS ones, is this upgrade method something that XS does not support?

Not that i'm aware of.

File: templates/iso/8.3/.treeinfo (hunk 1 above)
Comment (stormi) [reply templates/iso/8.3/.treeinfo:8]: Are they legacy if an upgrade path requires them?

Comment (stormi) [reply templates/iso/8.3/.treeinfo:8]: Ping.

**Comment by stormi** [comment]
> Those `.treeinfo` are modeled on the XS ones, is this upgrade method something that XS does not support?

They still have the code for it, but they don't mention it in their docs anymore. It's likely a feature that we are now alone to support. As long as it works (and we test it), it's an important upgrade path for users who can't easily boot an ISO.

**Review by stormi — APPROVED** [review]

**Pull request merged**